Deflake `agent/consul.TestAutopilot_BootstrapExpect`

This one has cropped up a number of times for me today. Its possible we changed something which made it happen more often.

## Example Logs

```
=== RUN   TestAutopilot_BootstrapExpect
--- FAIL: TestAutopilot_BootstrapExpect (8.27s)
    writer.go:29: 2020-02-04T18:53:08.098Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: Update: version=1
    writer.go:29: 2020-02-04T18:53:08.099Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.188Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.raft: initial configuration: index=0 servers=[]
    writer.go:29: 2020-02-04T18:53:08.188Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.raft: entering follower state: follower="Node at 127.0.0.1:22012 [Follower]" leader=
    writer.go:29: 2020-02-04T18:53:08.188Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.189Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.190Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.tlsutil: Update: version=1
    writer.go:29: 2020-02-04T18:53:08.191Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.191Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-6.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.191Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-6 (Addr: tcp/127.0.0.1:22012) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.278Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.raft: initial configuration: index=0 servers=[]
    writer.go:29: 2020-02-04T18:53:08.279Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.raft: entering follower state: follower="Node at 127.0.0.1:22015 [Follower]" leader=
    writer.go:29: 2020-02-04T18:53:08.279Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.280Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.280Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-7 (Addr: tcp/127.0.0.1:22015) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.280Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-7.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.280Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.tlsutil: Update: version=1
    writer.go:29: 2020-02-04T18:53:08.280Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.361Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.raft: initial configuration: index=0 servers=[]
    writer.go:29: 2020-02-04T18:53:08.361Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.raft: entering follower state: follower="Node at 127.0.0.1:22018 [Follower]" leader=
    writer.go:29: 2020-02-04T18:53:08.361Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.362Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.362Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-8 (Addr: tcp/127.0.0.1:22018) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.362Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-8.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.362Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: Update: version=1
    writer.go:29: 2020-02-04T18:53:08.363Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.393Z [WARN]  TestAutopilot_BootstrapExpect-node-6.server.raft: no known peers, aborting election
    writer.go:29: 2020-02-04T18:53:08.430Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: initial configuration: index=0 servers=[]
    writer.go:29: 2020-02-04T18:53:08.430Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: entering follower state: follower="Node at 127.0.0.1:22021 [Follower]" leader=
    writer.go:29: 2020-02-04T18:53:08.431Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.431Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.432Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-9.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.432Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-9 (Addr: tcp/127.0.0.1:22021) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.432Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22013
    writer.go:29: 2020-02-04T18:53:08.432Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:49658
    writer.go:29: 2020-02-04T18:53:08.433Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.433Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.433Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-7 (Addr: tcp/127.0.0.1:22015) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.433Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-6 (Addr: tcp/127.0.0.1:22012) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.434Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.434Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Stream connection from=127.0.0.1:56806
    writer.go:29: 2020-02-04T18:53:08.434Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.434Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22013
    writer.go:29: 2020-02-04T18:53:08.435Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:37362
    writer.go:29: 2020-02-04T18:53:08.435Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:49660
    writer.go:29: 2020-02-04T18:53:08.435Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-8 (Addr: tcp/127.0.0.1:22018) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-7.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.436Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.436Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.437Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-6 address=127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.437Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-6.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.437Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-7 (Addr: tcp/127.0.0.1:22015) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.437Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.438Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22013
    writer.go:29: 2020-02-04T18:53:08.438Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:49668
    writer.go:29: 2020-02-04T18:53:08.439Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.439Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.440Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.440Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.440Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.440Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-8 (Addr: tcp/127.0.0.1:22018) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.440Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.441Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-7 address=127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.442Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.442Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.442Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.443Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Found expected number of peers, attempting bootstrap: peers=127.0.0.1:22012,127.0.0.1:22015,127.0.0.1:22018
    writer.go:29: 2020-02-04T18:53:08.443Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.443Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.444Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.444Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Found expected number of peers, attempting bootstrap: peers=127.0.0.1:22018,127.0.0.1:22015,127.0.0.1:22012
    writer.go:29: 2020-02-04T18:53:08.470Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-6 (Addr: tcp/127.0.0.1:22012) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.472Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.472Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.472Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.472Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.472Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-8 (Addr: tcp/127.0.0.1:22018) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.472Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.473Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Found expected number of peers, attempting bootstrap: peers=127.0.0.1:22021,127.0.0.1:22018,127.0.0.1:22012,127.0.0.1:22015
    writer.go:29: 2020-02-04T18:53:08.480Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-9 (Addr: tcp/127.0.0.1:22021) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.481Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Stream connection from=127.0.0.1:56830
    writer.go:29: 2020-02-04T18:53:08.482Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.482Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.482Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.483Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.483Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.483Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22020
    writer.go:29: 2020-02-04T18:53:08.484Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.484Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.484Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.484Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.501Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Stream connection from=127.0.0.1:49370
    writer.go:29: 2020-02-04T18:53:08.501Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.501Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.502Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-7.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.502Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-6.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.504Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.504Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.504Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.504Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.506Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.506Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.507Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.507Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-8 address=127.0.0.1:22020
    writer.go:29: 2020-02-04T18:53:08.508Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-8.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.508Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.508Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.508Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22023
    writer.go:29: 2020-02-04T18:53:08.509Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.509Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-9 (Addr: tcp/127.0.0.1:22021) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.509Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.510Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.510Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-7 address=127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.510Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Existing Raft peers reported by server, disabling bootstrap mode: server=TestAutopilot_BootstrapExpect-node-6
    writer.go:29: 2020-02-04T18:53:08.510Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-9 (Addr: tcp/127.0.0.1:22021) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.510Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-8.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.510Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.511Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22023
    writer.go:29: 2020-02-04T18:53:08.511Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.514Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-6 (Addr: tcp/127.0.0.1:22012) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.514Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Adding LAN server: server="TestAutopilot_BootstrapExpect-node-7 (Addr: tcp/127.0.0.1:22015) (DC: dc1)"
    writer.go:29: 2020-02-04T18:53:08.526Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:37398
    writer.go:29: 2020-02-04T18:53:08.527Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-6 address=127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.528Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Stream connection from=127.0.0.1:36802
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-8.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-7.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-6.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-8.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-7.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.529Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-6.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.529Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Stream connection from=127.0.0.1:36798
    writer.go:29: 2020-02-04T18:53:08.530Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22020
    writer.go:29: 2020-02-04T18:53:08.530Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Stream connection from=127.0.0.1:49380
    writer.go:29: 2020-02-04T18:53:08.530Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.530Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-9 address=127.0.0.1:22023
    writer.go:29: 2020-02-04T18:53:08.530Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-9.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.530Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.530Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-9 address=127.0.0.1:22023
    writer.go:29: 2020-02-04T18:53:08.530Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-9.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.530Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: EventMemberJoin: TestAutopilot_BootstrapExpect-node-9.dc1 127.0.0.1
    writer.go:29: 2020-02-04T18:53:08.531Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-8 address=127.0.0.1:22020
    writer.go:29: 2020-02-04T18:53:08.531Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: Handled event for server in area: event=member-join server=TestAutopilot_BootstrapExpect-node-9.dc1 area=wan
    writer.go:29: 2020-02-04T18:53:08.531Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.531Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.memberlist: memberlist: Stream connection from=127.0.0.1:37404
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-6 address=127.0.0.1:22014
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.memberlist: memberlist: Initiating push/pull sync with: 127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Stream connection from=127.0.0.1:56848
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.532Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server: Successfully performed flood-join for server at address: server=TestAutopilot_BootstrapExpect-node-7 address=127.0.0.1:22017
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.533Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.562Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.562Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.562Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.563Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.563Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.563Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8.dc1
    writer.go:29: 2020-02-04T18:53:08.564Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.567Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.567Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.567Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.567Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.567Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:08.568Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.580Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.589Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.590Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.590Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.590Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.590Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.631Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.632Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-8
    writer.go:29: 2020-02-04T18:53:08.648Z [WARN]  TestAutopilot_BootstrapExpect-node-7.server.raft: no known peers, aborting election
    writer.go:29: 2020-02-04T18:53:08.656Z [WARN]  TestAutopilot_BootstrapExpect-node-9.server.raft: heartbeat timeout reached, starting election: last-leader=
    writer.go:29: 2020-02-04T18:53:08.656Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: entering candidate state: node="Node at 127.0.0.1:22021 [Candidate]" term=2
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-6.dc1
    writer.go:29: 2020-02-04T18:53:08.662Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-7.dc1
    writer.go:29: 2020-02-04T18:53:08.663Z [WARN]  TestAutopilot_BootstrapExpect-node-8.server.raft: heartbeat timeout reached, starting election: last-leader=
    writer.go:29: 2020-02-04T18:53:08.663Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.raft: entering candidate state: node="Node at 127.0.0.1:22018 [Candidate]" term=2
    writer.go:29: 2020-02-04T18:53:08.668Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.raft: votes: needed=3
    writer.go:29: 2020-02-04T18:53:08.668Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.raft: vote granted: from=127.0.0.1:22021 term=2 tally=1
    writer.go:29: 2020-02-04T18:53:08.668Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.raft: accepted connection: local-address=127.0.0.1:22015 remote-address=127.0.0.1:36086
    writer.go:29: 2020-02-04T18:53:08.668Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.raft: lost leadership because received a requestVote with a newer term
    writer.go:29: 2020-02-04T18:53:08.698Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.raft: votes: needed=2
    writer.go:29: 2020-02-04T18:53:08.698Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.raft: vote granted: from=127.0.0.1:22018 term=2 tally=1
    writer.go:29: 2020-02-04T18:53:08.699Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.raft: accepted connection: local-address=127.0.0.1:22012 remote-address=127.0.0.1:59698
    writer.go:29: 2020-02-04T18:53:08.699Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.raft: lost leadership because received a requestVote with a newer term
    writer.go:29: 2020-02-04T18:53:08.710Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.raft: accepted connection: local-address=127.0.0.1:22018 remote-address=127.0.0.1:45890
    writer.go:29: 2020-02-04T18:53:08.710Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.raft: duplicate requestVote for same term: term=2
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.raft: accepted connection: local-address=127.0.0.1:22015 remote-address=127.0.0.1:36094
    writer.go:29: 2020-02-04T18:53:08.711Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.raft: duplicate requestVote for same term: term=2
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.raft: accepted connection: local-address=127.0.0.1:22012 remote-address=127.0.0.1:59700
    writer.go:29: 2020-02-04T18:53:08.711Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.raft: vote granted: from=127.0.0.1:22015 term=2 tally=2
    writer.go:29: 2020-02-04T18:53:08.731Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.762Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.762Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.763Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageJoinType: TestAutopilot_BootstrapExpect-node-9.dc1
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.raft: duplicate requestVote for same term: term=2
    writer.go:29: 2020-02-04T18:53:08.791Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.raft: vote granted: from=127.0.0.1:22012 term=2 tally=3
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: election won: tally=3
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: entering leader state: leader="Node at 127.0.0.1:22021 [Leader]"
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: added peer, starting replication: peer=127.0.0.1:22018
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: added peer, starting replication: peer=127.0.0.1:22012
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: added peer, starting replication: peer=127.0.0.1:22015
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: cluster leadership acquired
    writer.go:29: 2020-02-04T18:53:08.791Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: New leader elected: payload=TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.791Z [WARN]  TestAutopilot_BootstrapExpect-node-7.server.raft: failed to get previous log: previous-index=1 last-index=0 error="log not found"
    writer.go:29: 2020-02-04T18:53:08.791Z [WARN]  TestAutopilot_BootstrapExpect-node-9.server.raft: appendEntries rejected, sending older logs: peer="{Voter 127.0.0.1:22015 127.0.0.1:22015}" next=1
    writer.go:29: 2020-02-04T18:53:08.792Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: pipelining replication: peer="{Voter 127.0.0.1:22012 127.0.0.1:22012}"
    writer.go:29: 2020-02-04T18:53:08.798Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.raft: entering follower state: follower="Node at 127.0.0.1:22018 [Follower]" leader=127.0.0.1:22021
    writer.go:29: 2020-02-04T18:53:08.798Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: pipelining replication: peer="{Voter 127.0.0.1:22015 127.0.0.1:22015}"
    writer.go:29: 2020-02-04T18:53:08.798Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: pipelining replication: peer="{Voter 127.0.0.1:22018 127.0.0.1:22018}"
    writer.go:29: 2020-02-04T18:53:08.812Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.raft: accepted connection: local-address=127.0.0.1:22015 remote-address=127.0.0.1:36096
    writer.go:29: 2020-02-04T18:53:08.831Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.raft: accepted connection: local-address=127.0.0.1:22018 remote-address=127.0.0.1:45902
    writer.go:29: 2020-02-04T18:53:08.832Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.raft: accepted connection: local-address=127.0.0.1:22012 remote-address=127.0.0.1:59706
    writer.go:29: 2020-02-04T18:53:08.832Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.832Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: New leader elected: payload=TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.832Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.832Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: New leader elected: payload=TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.832Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.832Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.832Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: New leader elected: payload=TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:08.861Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.tlsutil: OutgoingRPCWrapper: version=1
    writer.go:29: 2020-02-04T18:53:08.863Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.863Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.863Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.863Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.863Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.880Z [DEBUG] TestAutopilot_BootstrapExpect-node-6.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.880Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.880Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.880Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.896Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.897Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:08.897Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.serf: serf: messageUserEventType: consul:new-leader
    writer.go:29: 2020-02-04T18:53:09.006Z [DEBUG] connect.ca.consul: consul CA provider configured: id=07:80:c8:de:f6:41:86:29:8f:9c:b8:17:d6:48:c2:d5:c5:5c:7f:0c:03:f7:cf:97:5a:a7:c1:68:aa:23:ae:81 is_primary=true
    writer.go:29: 2020-02-04T18:53:09.173Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.connect: initialized primary datacenter CA with provider: provider=consul
    writer.go:29: 2020-02-04T18:53:09.173Z [INFO]  TestAutopilot_BootstrapExpect-node-9.leader: started routine: routine="CA root pruning"
    writer.go:29: 2020-02-04T18:53:09.173Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: member joined, marking health alive: member=TestAutopilot_BootstrapExpect-node-6
    writer.go:29: 2020-02-04T18:53:09.220Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: member joined, marking health alive: member=TestAutopilot_BootstrapExpect-node-7
    writer.go:29: 2020-02-04T18:53:09.271Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: member joined, marking health alive: member=TestAutopilot_BootstrapExpect-node-9
    writer.go:29: 2020-02-04T18:53:09.320Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: member joined, marking health alive: member=TestAutopilot_BootstrapExpect-node-8
    retry.go:121: autopilot_test.go:463: got 3 peers want 4
        
    writer.go:29: 2020-02-04T18:53:16.246Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server: shutting down server
    writer.go:29: 2020-02-04T18:53:16.246Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.leader: stopping routine: routine="CA root pruning"
    writer.go:29: 2020-02-04T18:53:16.246Z [WARN]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.246Z [DEBUG] TestAutopilot_BootstrapExpect-node-9.leader: stopped routine: routine="CA root pruning"
    writer.go:29: 2020-02-04T18:53:16.262Z [WARN]  TestAutopilot_BootstrapExpect-node-9.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.279Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.router.manager: shutting down
    writer.go:29: 2020-02-04T18:53:16.279Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: aborting pipeline replication: peer="{Voter 127.0.0.1:22018 127.0.0.1:22018}"
    writer.go:29: 2020-02-04T18:53:16.279Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: aborting pipeline replication: peer="{Voter 127.0.0.1:22015 127.0.0.1:22015}"
    writer.go:29: 2020-02-04T18:53:16.279Z [INFO]  TestAutopilot_BootstrapExpect-node-9.server.raft: aborting pipeline replication: peer="{Voter 127.0.0.1:22012 127.0.0.1:22012}"
    autopilot_test.go:415: TestAutopilot_BootstrapExpect-node-9 is shutdown
    writer.go:29: 2020-02-04T18:53:16.280Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server: shutting down server
    writer.go:29: 2020-02-04T18:53:16.280Z [WARN]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.307Z [WARN]  TestAutopilot_BootstrapExpect-node-8.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.316Z [INFO]  TestAutopilot_BootstrapExpect-node-8.server.router.manager: shutting down
    autopilot_test.go:415: TestAutopilot_BootstrapExpect-node-8 is shutdown
    writer.go:29: 2020-02-04T18:53:16.316Z [DEBUG] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Failed ping: TestAutopilot_BootstrapExpect-node-9.dc1 (timeout reached)
    writer.go:29: 2020-02-04T18:53:16.316Z [ERROR] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Failed to send indirect ping: write udp 127.0.0.1:22020->127.0.0.1:22017: use of closed network connection
    writer.go:29: 2020-02-04T18:53:16.316Z [ERROR] TestAutopilot_BootstrapExpect-node-8.server.memberlist: memberlist: Failed to send indirect ping: write udp 127.0.0.1:22020->127.0.0.1:22014: use of closed network connection
    writer.go:29: 2020-02-04T18:53:16.316Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server: shutting down server
    writer.go:29: 2020-02-04T18:53:16.317Z [WARN]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.317Z [WARN]  TestAutopilot_BootstrapExpect-node-7.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.330Z [DEBUG] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Failed ping: TestAutopilot_BootstrapExpect-node-9.dc1 (timeout reached)
    writer.go:29: 2020-02-04T18:53:16.330Z [ERROR] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Failed to send indirect ping: write udp 127.0.0.1:22017->127.0.0.1:22014: use of closed network connection
    writer.go:29: 2020-02-04T18:53:16.331Z [ERROR] TestAutopilot_BootstrapExpect-node-7.server.memberlist: memberlist: Failed to send indirect ping: write udp 127.0.0.1:22017->127.0.0.1:22020: use of closed network connection
    writer.go:29: 2020-02-04T18:53:16.344Z [INFO]  TestAutopilot_BootstrapExpect-node-7.server.router.manager: shutting down
    autopilot_test.go:415: TestAutopilot_BootstrapExpect-node-7 is shutdown
    writer.go:29: 2020-02-04T18:53:16.345Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server: shutting down server
    writer.go:29: 2020-02-04T18:53:16.345Z [WARN]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.362Z [WARN]  TestAutopilot_BootstrapExpect-node-6.server.serf: serf: Shutdown without a Leave
    writer.go:29: 2020-02-04T18:53:16.362Z [INFO]  TestAutopilot_BootstrapExpect-node-6.server.router.manager: shutting down
    autopilot_test.go:415: TestAutopilot_BootstrapExpect-node-6 is shutdown
```

https://circleci.com/gh/hashicorp/consul/124479#tests/containers/1

--- a/agent/consul/autopilot_test.go
+++ b/agent/consul/autopilot_test.go
@@ -399,7 +399,7 @@ func TestAutopilot_PromoteNonVoter(t *testing.T) {
 	})
 }
 
-func TestAutopilot_BootstrapExpect(t *testing.T) {
+func TestAutopilot_MinQuorum(t *testing.T) {
 	dc := "dc1"
 	closeMap := make(map[string]chan struct{})
 	conf := func(c *Config) {

--- a/agent/consul/autopilot_test.go
+++ b/agent/consul/autopilot_test.go
@@ -405,7 +405,7 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 	conf := func(c *Config) {
 		c.Datacenter = dc
 		c.Bootstrap = false
-		c.BootstrapExpect = 3
+		c.BootstrapExpect = 4
 		c.AutopilotConfig.MinQuorum = 3
 		c.RaftConfig.ProtocolVersion = raft.ProtocolVersion(2)
 		c.AutopilotInterval = 100 * time.Millisecond
@@ -427,7 +427,7 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 
 	dir3, s3 := testServerWithConfig(t, conf)
 	defer os.RemoveAll(dir3)
-	defer s3.Shutdown()
+	defer s3.Shutdown() 
 
 	dir4, s4 := testServerWithConfig(t, conf)
 	defer os.RemoveAll(dir4)
@@ -449,26 +449,31 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 			if mem.IsLeader() && leader {
 				return mem
 			}
-			if !leader && !mem.IsLeader() {
+			if !mem.IsLeader() && !leader {
 				return mem
 			}
 		}
 
-		t.Fatalf("no members set")
 		return nil
 	}
-
+	testrpc.WaitForLeader(t, s1.RPC, dc)
 	for _, s := range servers {
-		testrpc.WaitForLeader(t, s.RPC, dc)
 		retry.Run(t, func(r *retry.R) { r.Check(wantPeers(s, 4)) })
 	}
 
 	// Have autopilot take one into left
+
 	dead := findStatus(false)
+	if dead == nil {
+		t.Fatalf("no members set")
+	}
 	dead.Shutdown()
 	<-closeMap[dead.config.NodeName]
 	retry.Run(t, func(r *retry.R) {
 		leader := findStatus(true)
+		if leader == nil {
+			r.Fatalf("no members set")
+		}
 		for _, m := range leader.LANMembers() {
 			if m.Name == dead.config.NodeName && m.Status != serf.StatusLeft {
 				r.Fatalf("%v should be left, got %v", m.Name, m.Status.String())
@@ -479,11 +484,16 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 	delete(servers, dead.config.NodeName)
 	//Autopilot should not take this one into left
 	dead = findStatus(false)
-	dead.Shutdown()
+	if err := dead.Shutdown(); err != nil {
+		t.Fatalf("could not shut down %s, error %v", dead.config.NodeName, err)
+	}
 	<-closeMap[dead.config.NodeName]
 
 	retry.Run(t, func(r *retry.R) {
 		leader := findStatus(true)
+		if leader == nil {
+			r.Fatalf("no members set")
+		}
 		for _, m := range leader.LANMembers() {
 			if m.Name == dead.config.NodeName && m.Status != serf.StatusFailed {
 				r.Fatalf("%v should be failed, got %v", m.Name, m.Status.String())

--- a/agent/consul/autopilot_test.go
+++ b/agent/consul/autopilot_test.go
@@ -427,7 +427,7 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 
 	dir3, s3 := testServerWithConfig(t, conf)
 	defer os.RemoveAll(dir3)
-	defer s3.Shutdown() 
+	defer s3.Shutdown()
 
 	dir4, s4 := testServerWithConfig(t, conf)
 	defer os.RemoveAll(dir4)

--- a/agent/consul/autopilot_test.go
+++ b/agent/consul/autopilot_test.go
@@ -446,10 +446,10 @@ func TestAutopilot_MinQuorum(t *testing.T) {
 	//Differentiate between leader and server
 	findStatus := func(leader bool) *Server {
 		for _, mem := range servers {
-			if mem.IsLeader() && leader {
+			if mem.IsLeader() == leader {
 				return mem
 			}
-			if !mem.IsLeader() && !leader {
+			if !mem.IsLeader() == !leader {
 				return mem
 			}
 		}
@@ -457,9 +457,6 @@ func TestAutopilot_MinQuorum(t *testing.T) {
 		return nil
 	}
 	testrpc.WaitForLeader(t, s1.RPC, dc)
-	for _, s := range servers {
-		retry.Run(t, func(r *retry.R) { r.Check(wantPeers(s, 4)) })
-	}
 
 	// Have autopilot take one into left
 	dead := findStatus(false)

--- a/agent/consul/autopilot_test.go
+++ b/agent/consul/autopilot_test.go
@@ -462,7 +462,6 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 	}
 
 	// Have autopilot take one into left
-
 	dead := findStatus(false)
 	if dead == nil {
 		t.Fatalf("no members set")

